scripts/extract-package-code.R

```diff
@@ -126,3 +138,101 @@ extract_package_vignettes <- function(pkg, pkg_dir, output_dir) {
 
     vignettes
 }
+
+
+path_sanitized <- function(path) {
+    path_abs(path_tidy(path), getwd())
+}
+
+parse_command_line_arguments <- function() {
+
+    option_list <- list(
+        make_option(c("-r", "--r-dyntrace"), action="store", type="character",
+                    default="../../R-dyntrace/bin/R", help="R-dyntrace interpreter.",
+                    metavar="r-dyntrace"),
+        make_option(c("-v", "--verbose"), action="store_true", default=FALSE,
+                    help="Flag to enable verbose mode.", metavar="verbose"),
+        make_option(c("--corpus-dirpath"), action="store", type="character",
+                    help="Directory for storing vignettes.", metavar="corpus-dirpath"),
+        make_option(c("--raw-analysis-dirpath"), action="store", type="character",
+                    help="Directory for tracer generated data.", metavar="raw-analysis-dirpath"),
+        make_option(c("--disable-eval-expression-analysis"), action="store_true", default=FALSE,
+                    help="Flag to disable eval expression analysis.", metavar="eval-expression-analysis")
+    )
+
+    args <- parse_args(OptionParser(option_list = option_list),
+                       positional_arguments=TRUE)
+
+    list(package = args$args[1],
```

Comment:
Can't this be done automatically?
```r

options <- append(list(package=args$args[1]), args$options)
names(options) <- str_replace_all(names(options), "-", "-")
options
```

Reply:
We can but we will still have to santize 3 paths and change `disable-eval-expression-analysis` to `eval-expression-analysis`. At that point, the difference between the two approaches will go away.